Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'sinatra',         '~> 1.4.2'
 gem 'rake',            '~> 0.9.2.2'
 gem 'multi_json'
 
-gem 'sentry-raven',    github: 'getsentry/raven-ruby'
+gem 'sentry-raven',    git: 'https://github.com/getsentry/raven-ruby'
```

Comment:
Doing these PRs is already quite a lot of work as-is. That's why I'd rather do that update as a separate step if possible, allowing it to be handled at a different time by a different person.